src/cargo/util/toml/mod.rs

```diff
@@ -240,6 +241,9 @@ pub struct TomlManifest {
     patch: Option<BTreeMap<String, BTreeMap<String, TomlDependency>>>,
     workspace: Option<TomlWorkspace>,
     badges: Option<BTreeMap<String, BTreeMap<String, String>>>,
+    lints: Option<BTreeMap<String, String>>,
+    #[serde(rename = "lints2")]
```

Comment:
I'm not entirely clear on what the `lints2` name is doing here, can you add a comment to the naming choice here?

Reply:
Hm ok, I'm not really sure we'd want to support `cfg(..)` for lint configurations though in the sense that it feels like it's a bit overkill for configuration that can otherwise be specific in the crate source anyway?

Reply:
Right, let's simplify to just a `[lints]` section then.